js/wallet.js

```diff
@@ -56,10 +236,77 @@ class JS extends Wallet {
     mockery.disable()
   }
 
-  createWallet () {
+  afterEach (mockery) {
+    mockery.resetCache()
+    this.setStateFile()
+    this.state = defaultAppState
+    this.ledger.resetModules()
+  }
+
+  getInfo (state) {
+    return state.getIn(['ledger', 'info'])
+  }
+
+  setInfo (state, prop, value) {
+    return state.setIn(['ledger', 'info', prop], value)
```

Comment:
Maybe you could use `ledgerState.getInfoProp`

Reply:
++